src/containers/Settings/types.ts

```diff
@@ -0,0 +1,31 @@
+import { Action } from 'redux'
```

Comment:
this file is deprecated and should not be added again. Use `useConfig` from `@apisuite/fe-base` to access settings.